src/datasets/jetstream.md

```diff
@@ -0,0 +1,243 @@
+Statistical summaries of telemetry data from experiments run in Mozilla
+products are provided by Jetstream. These summaries are published to
+BigQuery and serve both as the substrate for the result visualization
+platform and as a resource for data scientists.
+
+Jetstream runs as part of the nightly ETL job. The tables are published
```

Comment:
Generally for datasets we put the scheduling information at the bottom in a seperate section called "scheduling":

https://docs.telemetry.mozilla.org/datasets/batch_view/clients_daily/reference.html#scheduling

Can you use also provide a link to whatever code (in telemetry-airflow) which is doing the scheduling?

Reply:
I added the scheduling system with the tech specs but I want to highlight this to data scientists early.